### ISSUE TITLE
The website is dead

### ISSUE THREAD

**Issue by SandeMC** [magic-ike/spotify-data-card#1]

**Comment by magic-ike** [comment]
That's strange, @SandeMC. There was some downtime on Saturday due to me switching domain registrars, but the site should be up now, and everything looks fine for me. Try [hard refreshing](https://www.contractsafe.com/support/how-to-clear-your-browser-cache-and-hard-refresh) on the site to clear out your cache and let me know if that fixes the issue.

**Comment by kreatoo** [comment]
Can reproduce the issue.
![image](https://github.com/magic-ike/spotify-data-card/assets/89093869/53a36957-5d47-412e-9445-00c894728b9a)

**Comment by henrygame3** [comment]
![Screenshot_20230705-020526~2](https://github.com/magic-ike/spotify-data-card/assets/29360012/cd2598a9-af56-40ee-9d25-01e1a312836f)
Doesn't work here either

**Comment by magic-ike** [comment]
I'm still not able to reproduce on multiple devices, @kreatoo. Can anyone confirm that the issue persists after a hard refresh?

**Comment by kreatoo** [comment]
@magic-ike did you try a different network?  I tried hard-refreshing yet i still have the same issue. I also never visited the site on this browser before, so there was no cache involved.

**Comment by magic-ike** [comment]
> @magic-ike did you try a different network? I tried hard-refreshing yet i still have the same issue. I also never visited the site on this browser before, so there was no cache involved.

Ah, I didn't try on a different network so I may be able to reproduce that way. That said, is this link also broken or no? https://data-card-for-spotify.herokuapp.com/

**Comment by kreatoo** [comment]
For some reason, it works on my mobile phone thru cellular network.
I still don't get why doesn't it work on my home network though.
@SandeMC also has the same issue, and our networks are different.

**Comment by kreatoo** [comment]
> > @magic-ike did you try a different network? I tried hard-refreshing yet i still have the same issue. I also never visited the site on this browser before, so there was no cache involved.
> 
> Ah, I didn't try on a different network so I may be able to reproduce that way. That said, is this link also broken or no? https://data-card-for-spotify.herokuapp.com/

This link works perfectly.

**Comment by magic-ike** [comment]
Ok, my current theories are either a) somehow my old domain registrar is blocking the main domain (not sure if this is even possible so this may not be likely) or b) there's some domain-related server-side caching on your networks that needs some time to refresh (hard refreshing on the browser won't fix this, unfortunately). That said, I'll see if I can reproduce on a different network.

**Comment by magic-ike** [comment]
@SandeMC @kreatoo I was able to reproduce the issue on different networks. I'm still not sure exactly what the issue is but there's a possibility that the *data-card-for-spotify.com* domain name is being blocked by certain ISPs (or some other service provider(s)) due to its similarities to the *spotify.com* domain name. Whether or not this is the case, I think I want to retire that domain name anyway due to the "unsafe site" warning it triggers on Chrome (for the same reason). Since the Heroku one appears to be working just fine, I'm just going to make that the new official domain name for the service (the README has been updated to reflect this), and I'd recommend you just use that one moving forward.

Hope this resolves the issue. Happy 4th! (if you're American 🇺🇸)
